### PR TITLE
feat(sdk-coin-bera): add bera token support

### PR DESCRIPTION
**Pull request by adarsh312**
Ticket: WIN-3493

BGT is a governance token for BERA, it is based on the ERC20 standard

### CODE REVIEW

**Review by OttoAllmendinger — COMMENTED** [review]
Automatic comment to flush this PR from my review queue. 🚽🚽🚽
Please re-request another review from the respective team(s) when you are ready for reviews.

**Comment by Vijay-Jagannathan** [comment]
flush

**Review by Vijay-Jagannathan — COMMENTED** [review]
flush

**Review by zahin-mohammad — COMMENTED** [review]
pr is in draft

File: modules/statics/src/account.ts [hunk 1]
```diff
@@ -1998,6 +2008,96 @@ export function tzkethErc20(
...
+      asset,
+      isToken: true,
+      primaryKeyCurve,
+      baseUnit: BaseUnit.ETH,
+    })
+  );
+}
+
+/**
+ * Factory function for zkSync Sepolia testnet beraErc20 token instances.
```
Comment (gianchandania) [inline modules/statics/src/account.ts:2059]: nit: zksync?

**Review by gianchandania — APPROVED** [review]

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -17966,6 +17968,23 @@ export const coins = CoinMap.fromCoins([
     '0xcccb29bac5ad81290383643c6fb38130cda9d881',
     UnderlyingAsset['tzketh:link']
   ),
+  beraErc20(
+    'ef833f4e-7617-4c6d-8a1f-1fef0dd1dd0e',
+    'bera:bgt',
+    'BGT Token',
+    18,
+    // TODO: the mainnet contract address is still not available, adding placeholder here
```
Comment (zahin-mohammad) [inline modules/statics/src/coins.ts:17976]: Add a ticket for this TODO.

**Review by zahin-mohammad — APPROVED** [review]

**Comment by zahin-mohammad** [comment]
Update codeowners please for this new sdk module.

File: modules/statics/src/account.ts (hunk 1 above)
Comment (adarsh312) [reply modules/statics/src/account.ts:2059]: missed it, i will fix it

**Pull request merged**